main.go

```diff
@@ -62,6 +62,11 @@ func main() {
 					Name:  "user",
 					Usage: "Return given user public ssh keys (this option can be used multiple times for multiple users)",
 				},
+				cli.StringFlag{
+					Name:  "comment",
+					Usage: "Include comment",
+					Value: "Generated by fetch-ssh-keys - DO NOT EDIT",
```

Comment:
Should we default to not write the comment? And if you want some comment, you add the —comment flag. So if users upgrade and keep the old flags, the functionality stays same.

Reply:
I've been having the same thought after submitting the PR. I guess that'll make it a good idea :-)

Stay tuned.